hail/src/main/scala/is/hail/io/bgen/LoadBgen.scala

```diff
@@ -341,15 +340,15 @@ case class MatrixBGENReader(
   includedVariants: Option[TableIR]) extends MatrixHybridReader {
   private val hc = HailContext.get
   private val sc = hc.sc
-  private val hConf = sc.hadoopConfiguration
+  private val fs = hc.sFS
```

Comment:
I don't know why sc.hadoopConfiguration was referenced instead of hc.hadoopConf...spoke to @tpoterba, who indicated that as there is a single sparkContext, there should be a single hadoopConfiguration (there appears to be only one exception to this, in tests).